components/tooltip/tooltip.js

```diff
@@ -892,19 +909,22 @@ class Tooltip extends RtlMixin(LitElement) {
 	}
 
 	async _showingChanged(newValue, dispatch) {
+		if (!this.isConnected) return;
+
 		clearTimeout(this._hoverTimeout);
 		clearTimeout(this._longPressTimeout);
+
 		if (newValue) {
 			if (!this.forceShow) {
-				if (activeTooltip) activeTooltip.hide();
+				activeTooltip?.hide();
 				activeTooltip = this;
 			}
-
 			this._dismissibleId = setDismissible(() => this.hide());
 			this.setAttribute('aria-hidden', 'false');
 			await this.updateComplete;
-			await this.updatePosition();
-			if (dispatch) {
+
+			if (this._showing && dispatch) {
+				await this.updatePosition();
 				this.dispatchEvent(new CustomEvent(
```

Comment:
If, after the update, another tooltip has become active (and hidden this one), don't fire the `d2l-tooltip-show` event since it never actually showed. Also, don't both updating the position since it can and should be recalculated on the next `show`